release/models/gnpsi/openconfig-gnpsi-types.yang

```diff
@@ -33,6 +33,6 @@ module openconfig-gnpsi-types {
   identity GNPSI {
     base oc-grpct:GRPC_SERVICE;
```

Comment:
As it stands today, this identity is not used whatsoever due to the base identity not having an identityref

The transition for the current `services` leaf-list is a backwards incompatible change when this does occur